packages/web/src/dev/ValidatorsListBadges.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import { View, StyleSheet } from 'react-native'
+import { I18nProps, withNamespaces } from 'src/i18n'
+
+import * as lFounder from 'validator-badges/001-stake-off-founder-validator.md'
+import * as lAttestation from 'validator-badges/002-stake-off-attestation-maven.md'
+import * as lMaster from 'validator-badges/003-stake-off-master-validator.md'
+import * as lGenesis from 'validator-badges/004-genesis-validator.md'
+import * as lProposer from 'validator-badges/005-genesis-1st-proposer.md'
+import * as lTransaction from 'validator-badges/006-genesis-1st-transaction.md'
+import * as lVote from 'validator-badges/007-celo-foundation-vote-recipient.md'
+
+const allBadges = [
+  { src: '/images/badges/1st-proposer.svg', list: lProposer },
+  { src: '/images/badges/1st-transaction.svg', list: lTransaction },
+  { src: '/images/badges/attestation.svg', list: lAttestation },
+  { src: '/images/badges/founder-validator.svg', list: lFounder },
+  { src: '/images/badges/genesis-validator.svg', list: lGenesis },
+  { src: '/images/badges/master-validator.svg', list: lMaster },
+  { src: '/images/badges/vote-recipient.svg', list: lVote },
+].map((_) => ({ ..._, list: (_.list as any).toLowerCase() as string }))
```

Comment:
please give this variable a name. `_` should only be used in cases where the variable is not used.

Reply:
Also I believe `list` can be transformed into a Set like 

`.map((addresses) => ({ ...addresses, list:  new Set(addresses.split('/n')) }))`

this way below you can do 

 `allBadges.filter(({ list }) => list.has(address.toLowerCase()))`

and avoid looping inside a loop while rendering.

Reply:
`allBadges.filter(({ list }) => list.includes(address.toLowerCase()))`
This part of the code is not a double loop, is just a string matching. List is a long string with addresses (and new lines), so just checking if the string contains the address.

But anyway, I will change it.